src/forms/FormFieldMapCoordinates/MapCoordinatesDrawer/MapCoordinatesDrawer.tsx

```diff
@@ -36,44 +37,48 @@ const MapCoordinatesDrawer = (props: MapCoordinatesDrawerProps): ReactElement =>
 		dialogOpen,
 		handleDialogClose,
 	} = props;
+	const [isDragging, setIsDragging] = useState(false);
 
-	const modalReducer = useForm();
+	const { state, dispatch } = useForm();
 
 	const initialState = { lat: value.lat ? value.lat : undefined, lng: value.lng ? value.lng : undefined };

```

Comment:
Not sure if this is the correct place to add this but I have some notes:
1. When using the "with address property" the static map shows a marker but when you click on edit it doesn't, so I feel we should be adding a marker to that same position.
![image](https://user-images.githubusercontent.com/23409284/185021699-6b322b40-4233-4f02-a777-58edfb0b4d37.png)

2.  Doesn't matter if we're using the "withAddress" or just a regular save-and-edit, whenever you click on edit, it opens on the defaultLocation (mapPosition) I feel if we have a marker the map should open in that position.

3. Let's see if this is possible... I feel the map should re-center just when adding the coordinates manually in the fields, because it feels weird to me as a user that I add the coordinates but I have to zoom all the way out to see where the marker got located.